org-roam-capture.el

```diff
@@ -390,11 +426,18 @@ INFO is a plist for filling up Org-roam's capture templates.
 NODE is an `org-roam-node' construct containing information about the node.
 PROPS is a plist containing additional Org-roam properties for each template.
 TEMPLATES is a list of org-roam templates."
-  (let* ((props (plist-put props :call-location (point-marker)))
+  (let* ((props (thread-first props
+                  (plist-put :call-location (point-marker))
+                  (plist-put :action  (or (plist-get props :action) (if goto 'goto 'capture)))
```

Comment:
what about templates that are for both goto and capture? I think there should be an `'any` keyword support here as well.

Reply:
You mean declared in `org-roam-capture-templates` as `:action (capture goto)` or `:action any`? Or you mean the fallback behavior if it's not declared in the template? In the first case it should just work, while in the latter case I'm planning to fix it (was mentioned in the todo list).